Description: Disable broken, unmaintained dep serde-xml-rs
 .. which README.md itself states: "the broken XML and JSON renderers"
Forwarded: not-needed
Last-Update: 2024-09-11
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -29,4 +29,4 @@
 
-[dependencies.serde-xml-rs]
-version = "0.3.1"
+#[dependencies.serde-xml-rs]
+#version = "0.3.1"
 
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -16,3 +16,3 @@
 
-pub fn render_xml<W>(document: &Document, stream: W) -> Result<(), Error>
+/*pub fn render_xml<W>(document: &Document, stream: W) -> Result<(), Error>
 where
@@ -23,3 +23,3 @@
     Ok(())
-}
+}*/
 
